server/server.js

```diff
@@ -11,3 +11,27 @@
 // iterating through that list, display the string you built up.
 
 // Remember that the response is a stream object that must be closed.
+var http = require('http');
+var trucks = require('./trucks');
+var enhancedDate = require('./enhancedDate');
+var dayName = enhancedDate.getDayName;
+var monthName = enhancedDate.getMonthName;
+
+var availableTrucks = trucks.filterByDay(dayName);
```

Comment:
I invoked both getDayName and getMonthName.  Added date to the string correctly.  Also changed the trucks module to return an object with a filterByDay method.  I think.